.github/workflows/buildkit-daily.yml

```diff
@@ -0,0 +1,95 @@
+name: Publish buildkit daily container
+
+on:
+  schedule:
+    - cron: '0 17 * * *'
+  workflow_dispatch:
```

Comment:
I don't recall clearly but I [personally experienced](https://github.com/outloudvi/mw2fcitx/commit/bfad233133659e5f3e2a6728a9c3746c53893f0b) that setting `workflow_dispatch: null` might cause some problems, like there will be no button on the Actions page to trigger this. Please validate it.

Reply:
Leaving it blank has worked well for multiple workflows I own... Docs says so: https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#example